contract/src/main/java/com/ritense/valtimo/contract/json/patch/JsonPatchBuilder.java

```diff
@@ -144,4 +145,22 @@ public JsonPatch build() {
         return new JsonPatch(operations);
     }
 
+    private JsonNode get(JsonNode json, JsonPointer path) {
```

Comment:
I don't see any changes to `JsonPatchBuilderTest`, so I'm assuming the logic here is not tested.

Reply:
changed